src/navigation/config.js

```diff
@@ -369,7 +369,7 @@ export const explainSheetV2Config = {
   options: ({ route: { params = {} } }) => ({
     ...buildCoolModalConfig({
       ...params,
-      longFormHeight: params.sheetHeight,
+      longFormHeight: params.sheetHeight ?? -safeAreaInsetValues.bottom,
```

Comment:
don't want to show the sheet until we set the height. on ios, the bottom margin of the cool modal is above the bottom safe area inset (unlike android, where it's actually the true screen bottom)

Reply:
You can get rid of `params.sheetHeight` in both places now too 👍